FBC/FinancialInstruments/InstrumentPricing.rdf

```diff
@@ -411,7 +413,60 @@
 		</rdfs:subClassOf>
 		<rdfs:label xml:lang="en">security price</rdfs:label>
-		<skos:definition xml:lang="en">monetary price that some party is willing to pay, has recently paid, or would like to be paid for a given financial instrument at some point in time</skos:definition>
+		<skos:definition xml:lang="en">monetary price for a financial instrument at some point in time</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">A security price may be the price that some party is willing to pay, has recently paid, or would like to be paid, depending on the circumstances.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fbc-fi-ip;TradingDay">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-dt-fd;ExplicitDatePeriod"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-dt-fd;hasDuration"/>
+				<owl:hasValue rdf:resource="&fibo-fnd-dt-fd;Day"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-dt-fd;hasClosingDateTime"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+				<owl:onDataRange rdf:resource="&fibo-fnd-dt-fd;CombinedDateTime"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-dt-fd;hasOpeningDateTime"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+				<owl:onDataRange rdf:resource="&fibo-fnd-dt-fd;CombinedDateTime"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">trading day</rdfs:label>
+		<skos:definition xml:lang="en">time span that a particular trading venue is open</skos:definition>
+		<fibo-fnd-utl-av:abbreviation xml:lang="en">RTH</fibo-fnd-utl-av:abbreviation>
+		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">https://www.lawinsider.com/dictionary/trading-day</fibo-fnd-utl-av:adaptedFrom>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">In the United States, and with respect to common stock in particular, trading day means any day on which the stock is traded on the principal market, or, if the principal market is not the principal trading market for the common stock, then on the principal securities exchange or securities market on which the common stock is then traded, provided that &apos;Trading Day&apos; shall not include any day on which the common stock is scheduled to trade on such exchange or market for less than 4.5 hours or any day that the common stock is suspended from trading during the final hour of trading on such exchange or market (or if such exchange or market does not designate in advance the closing time of trading on such exchange or market, then during the hour ending at 4:00:00 p.m., New York time) unless such day is otherwise designated as a trading day in writing by the holder.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
Might be worth a note saying that by convention a value for hasOpeningDateTme is sufficient without the (redundant) hasClosingDateTime

Reply:
done